### PR TITLE
Pi 1880 mas overview UI

### PR DESCRIPTION
**Pull request by pmcphee77**

### CODE REVIEW

**Review by marcus-bcl — COMMENTED** [review]
Looks great! Just added a couple of minor questions/comments

File: server/views/pages/overview.njk [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+                    {%- default -%}
+                        {{ overview.previousOrders.breaches }} breaches on previous orders
+                {%- endswitch -%}
+                {% endset %}
+                {{ total}} ({{ breaches }})
+            {% endif %}
+        {% endset %}
+        {% set compliance %}
+            {% if overview.compliance %}
+                {{ overview.compliance.failureToComplyInLast12Months }} failure to comply within 12 months </br>
```
[the remark this answers is not in the document]
Comment (pmcphee77) [reply server/views/pages/overview.njk]: Have changed to <br/> for now will look in later PR to change

File: server/views/pages/index.njk [hunk 2]
```diff
@@ -4,8 +4,15 @@
...
 {% block content %}
-
-  <h1>This site is under construction...</h1>
-  <p>Please check back later when there is content to view.</p>
-
+    <h1 class="govuk-heading-l">Manage a Supervision</h1>
+    <div class="govuk-grid-column-two-thirds">
+        <p>Not all cases are suitable for the Manage supervisions pilot. This service is only suitable for cases that have:</p>
+        <ul class="govuk-list govuk-list--bullet">
+            <li>a test single community order</li>
```
[the remark this answers is not in the document]
Comment (pmcphee77) [reply server/views/pages/index.njk]: changed

File: server/views/_components/summary-card/macro.njk [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{% macro appSummaryCard(params) %}
```
[the remark this answers is not in the document]
Comment (pmcphee77) [reply server/views/_components/summary-card/macro.njk:1]: Yeah it looks a bit different so thought id use the macro they used in the prototype for now. Again we can investigate later if thats ok

File: server/routes/case.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+    await auditService.sendAuditMessage({
+      action: 'VIEW_MAS_SCHEDULE',
+      who: res.locals.user.username,
+      subjectId: crn,
+      subjectType: 'CRN',
+      correlationId: v4(),
+      service: 'hmpps-manage-a-supervision-ui',
+    })
+    const [overview] = await Promise.all([masClient.getOverview(crn)])
```
[the remark this answers is not in the document]
Comment (pmcphee77) [reply server/routes/case.ts]: done

File: server/views/pages/index.njk (hunk 2 above)
[the remark this answers is not in the document]
Comment (pmcphee77) [reply server/views/pages/index.njk]: done

**Review by marcus-bcl — APPROVED** [review]

**Pull request merged**